examples/topic_extraction_example.py

```diff
@@ -0,0 +1,98 @@
+"""Copyright 2019 REV
```

Comment:
Do you need to add LICENSE to these example files? It can be difficult to maintain them all. Can we just rely on the top level LICENSE?

Reply:
I'm not really sure. Maybe @vikram-rev would know

Reply:
For all licensing questions, I refer to legal.

Reply:
Who on legal do you normally ask?

Reply:
AFAIK it's generally recommended to add a license to all files and also at the top-level. There are tools that do it for you automatically, including in VScode. See https://opensource.google/documentation/reference/releasing/preparing#license-headers for a more detailed discussion. +1 to @amikofalvy suggestion to check with a lawyer as first preference though.